### PR TITLE
Add njal.la.xml

### PR DESCRIPTION
**Pull request by pipboy96**

### CODE REVIEW

File: src/chrome/content/rules/Njal.la.xml [hunk 1]
```diff
@@ -1,26 +0,0 @@
-<!--
-	Non-functional subdomains:
-		- mailfwd	(r)
-		- mx	(r)
-		- support	(r)
-		- test	(r)
-		- z	(r)
```
Comment (J0WI) [inline src/chrome/content/rules/Njal.la.xml]: Please keep comments that are still  true.

Comment (pipboy96) [reply src/chrome/content/rules/Njal.la.xml]: Will retest.

Comment (pipboy96) [reply src/chrome/content/rules/Njal.la.xml]: Will also run Amass.

Comment (pipboy96) [reply src/chrome/content/rules/Njal.la.xml]:
```
njal.la
z.njal.la
www.njal.la
netbox.njal.la
support.njal.la
r.njal.la
redirect.njal.la
ks.njal.la
git.njal.la
apt.njal.la
ns0.njal.la
mailfwd.njal.la
dns.njal.la
test.njal.la
xmpp.njal.la
ns4.njal.la
ns5.njal.la
vips.njal.la
ns1.njal.la
ns2.njal.la
ns3.njal.la
mx.njal.la
_xmpp-server._tcp.njal.la
_xmpp-client._tcp.njal.la
_jabber._tcp.njal.la
```

Comment (pipboy96) [reply src/chrome/content/rules/Njal.la.xml]: Ran Sublist3r in brute mode, there's more:
```
njal.la
www.njal.la
anycast.njal.la
approve.njal.la
apt.njal.la
dns.njal.la
git.njal.la
ks.njal.la
mailfwd.njal.la
mx.njal.la
netbox.njal.la
ns0.njal.la
ns1.njal.la
ns2.njal.la
ns3.njal.la
ns4.njal.la
ns5.njal.la
r.njal.la
redirect.njal.la
self.njal.la
support.njal.la
test.njal.la
vips.njal.la
xmpp.njal.la
z.njal.la
```

**Pull request closed without merging**